src/components/TextField/OfficeTextField.types.ts

```diff
@@ -0,0 +1,98 @@
+import {IStyle, IStyleFunctionOrObject} from "@uifabric/merge-styles";
+import {ITheme} from "@/styling";
+
+export interface ITextFieldSubComponentStyles {
```

Comment:
Please rename all the types to "IOfficeText...", that's what I did for the other ones aswell